source/features/extend-status-labels.tsx

```diff
@@ -8,24 +8,33 @@ function init(): false | void {
 		return false;
 	}
 
+	// Get the latest status update
 	const lastActionRef = select.all(`
-		.discussion-item-closed [href*="/pull/"],
-		.discussion-item-closed code,
+		.discussion-item-closed,
 		.discussion-item-reopened,
-		.discussion-item-merged [href*="/commit/"]
+		.discussion-item-merged
 	`).pop();
 
-	// Leave if it was never closed or if it was reopened
+	// Leave if the issue/PR was never closed or if it was reopened
```

Comment:
Changed the wording here because the "it" may be confusing in regards to the comment I added in line 11.